### PR TITLE
chore!: remove duplicate non-hexlified event attributes.

### PR DESCRIPTION
**Pull request by DimitrisJim**
## Description

closes: #5990 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against the correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to GitHub issue with discussion and accepted design, OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [conventional commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to follow the repository standards.
- [ ] Include a descriptive changelog entry when appropriate. This may be left to the discretion of the PR reviewers. (e.g. chores should be omitted from changelog)
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer.
- [ ] Review `SonarCloud Report` in the comment section below once CI passes.

## Summary by CodeRabbit

- **Refactor**
	- Updated packet data handling to use hexadecimal encoding, improving data consistency and readability.
- **Bug Fixes**
	- Deprecated older attributes to streamline event handling processes.
- **Tests**
	- Enhanced testing for packet and acknowledgment data handling to ensure reliability in hexadecimal format decoding.

### CODE REVIEW

**Comment by faddat** [comment]
nice one, thank you!

**Review by crodriguezvega — APPROVED** [review]
Looks good to me. Thanks for opening the PR to interchaintest.

Once we merge this to main, should we trigger a run of the `main` compatibility tests just to check nothing breaks?

And we should add an entry in the migration docs for this (fine if you add the link to the PR for now and we can write the details later on).

File: e2e/go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module github.com/cosmos/ibc-go/e2e
 
 go 1.21
 
+replace github.com/strangelove-ventures/interchaintest/v8 => github.com/DimitrisJim/interchaintest/v8 v8.0.0-20240419095404-2c9270423b9a
```
Comment (damiannolan) [inline e2e/go.mod:5]: Is the fork necessary or is there a commit on interchaintest which we can point at?

Would be good to add a comment either way for what the diff is and/or when we can remove it

**Review by damiannolan — APPROVED** [review]
Nice one!

**Comment by damiannolan** [comment]
Do we wanna changelog this?

**Review by colin-axner — APPROVED** [review]
ACK 

Note: this can be backported as a patch release if desired

**Pull request merged**